### PR TITLE
src/components: Update styling for HelpButton component

### PR DESCRIPTION
**Pull request by maceksimon**
Update styling for `HelpButton` component based on design.

* Use `slot` for more flexibility in button styling.
* Add custom icon.
* Update usage in `LoginRegisterHeader` (with default button).
* Update usage in `DrawerHeader` (with slot).
* Update button and text colors in the embedded `ContactForm` component.
* Update and refactor tests (with selectors and variables).

### CODE REVIEW

File: src/components/global/DrawerHeader.vue [hunk 1]
```diff
@@ -76,12 +76,30 @@ export default defineComponent({
...
+            unelevated
+            round
+            color="primary"
+            size="8px"
+            @click.prevent="openDialog"
+            data-cy="button-help"
+          >
+            <q-icon
+              name="svguse:/icons/custom.svg#question-mark"
+              size="18px"
```
Comment (tmszi) [inline src/components/global/DrawerHeader.vue:91]: `"18px"` repeat

**Comment by tmszi** [comment]
Better icon file name [public/icons/custom.svg](https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/560/files#diff-53a21d26f364b9df4114ae3b1114f9485e943b1aecbd88f958b1ea7ac14d677c) 

If file contains icons for button widget, file name would be button_icons.svg.

File: src/components/global/DrawerHeader.vue (hunk 1 above)
Comment (maceksimon) [reply src/components/global/DrawerHeader.vue]: Fixed.

**Comment by maceksimon** [comment]
> Better icon file name [public/icons/custom.svg](https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/560/files#diff-53a21d26f364b9df4114ae3b1114f9485e943b1aecbd88f958b1ea7ac14d677c)
> 
> If file contains icons for button widget, file name would be button_icons.svg.

Updated.

**Review by tmszi — APPROVED** [review]

**Pull request merged**